docker/base/Dockerfile

```diff
@@ -15,6 +15,36 @@
 # NOTE: We try to follow the guide in https://docs.docker.com/develop/develop-images/dockerfile_best-practices/
 #       Each RUN contains a bundle of steps, which reduces the cache.
 
+from golang:1.19 AS go_builder
+
+RUN set -eux; \
+    echo "iteration 0"; \
+    git clone --branch aws_nitro https://github.com/dreemkiller/services.git --branch v0.2 --depth 1; \
+    git clone https://github.com/veracruz-project/proxy_attestation_server.git --branch v0.1 --depth 1; \
+    cd proxy_attestation_server; \
+    go build -o ./vts github.com/veraison/services/vts/cmd/vts-service; \
```

Comment:
I should have been clearer that the comment applied to the repositories involved in the `go build` steps as well.

Reply:
Those are currently not pulling from github, but from the `services` pull above them (there's an override in the `go.mod` file as not all of the necessary changes have been upstreamed). Once we get them upstreamed, the `git clone` for `services` will no longer be necessary, and the `go.mod` file will specify the exact revision needed.